packages/paste-website/src/pages/blog/2021-07-26-pastes-path-to-a-transparent-package-categorization-system.mdx

```diff
@@ -96,12 +74,11 @@ Rest assured, all of the components and patterns currently in the system are _pr
 
 After making the switch to align our definitions of _alpha_, _beta_ and _production_ to indicate level of risk, we needed a way to transparently communicate which steps each component/pattern had completed. Rather than creating a new categorization system, we decided to just, well, tell you. You can now find a checklist of all packages on the [Components Overview](/components) and [Patterns Overview](/patterns) pages, as well as individual package status checklists at the top of each page of documentation.
 
-<Box marginBottom="space70">
-  <Img
-    fluid={props.data.overview2.childImageSharp.fluid}
-    alt="Screenshot of the Component Overview page after we updated our categorization system"
-  />
-</Box>
+<contentspacer>
```

Comment:
question: what is this component? I haven't encountered it before. Is there a reason it's used here instead of Box?

Reply:
It new. It is just the same box, but as it's added to the MDX provider, you can now just place markdown inside of it.

I wanted to simplify the use of images as much as possible, and where ever you use a component in mdx you can no longer use markdown inside of it. 

If you add it to the MDX provider, essentially making it a shortcode (but not quite) MDX knows it's a components, and allows normal markdown. 

It just really bugs me when I see html and components in these MDX pages for just text content because we used a callout or a do dont